docs/notebooks/noise.ipynb

```diff
@@ -0,0 +1,172 @@
+
```

Comment:
@Strilanc This is the notebook that is "broken" by removing a `{` in the beginning. If you try to open view the file, you get:
<img width="828" alt="Screen Shot 2019-11-21 at 9 29 04 AM" src="https://user-images.githubusercontent.com/1130416/69361465-67141b80-0c41-11ea-92a1-337ae8c4c997.png">

Travis passed, so this type of error is not being caught. What should we do?

Reply:
Open an issue to write tooling that tests the notebooks.